modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -1028,6 +1042,39 @@ else if (insertIndex == existingMethods.size()) {
 		return newFileContentSB.toString();
 	}
 
+	private boolean _isCommentLine(String content, int position) {
+		int lineNumber = getLineNumber(content, position);
+
+		String line = getLine(content, lineNumber);
+
+		line = line.trim();
+
+		if (line.startsWith("//") || line.startsWith("<%--")) {
+			return true;
+		}
+
+		int lastBlockStart = content.lastIndexOf("/*", position);
```

Comment:
@nicolas-sss, can we rename the variables here to `start` and `end`, and reuse those variables?